.gitignore

```diff
@@ -180,3 +180,5 @@ TempTypeSpecFiles/
 
 # Common toolchain intermediate files
 temp
+
```

Comment:
You should not be making changes to root files.   Please revert.

Reply:
thanks, done